core/modules/widgets/select.js

```diff
@@ -53,6 +53,17 @@ Handle a change event
 SelectWidget.prototype.handleChangeEvent = function(event) {
 	var value = this.getSelectDomNode().value;
 	this.wiki.setText(this.selectTitle,this.selectField,this.selectIndex,value);
+
+	this.activeChild = 2*(this.getSelectDomNode().selectedIndex)+1;
```

Comment:
What is this calculation good for?
I didn't check the code, but this seems to be not generic.